addons/misra.py

```diff
@@ -223,7 +223,7 @@ def isUnsignedInt(expr):
     if not expr:
         return False
     if expr.isNumber:
-        return expr.str.find('u') > 0 or expr.str.find('U') > 0
+        return 'u' in expr.str or 'U' in expr.str
```

Comment:
I guess this is ok.. the 'u' and 'U' should not be first char. but since `expr.isNumber` is true, we know that first char is not a 'u' or 'U'.